verifier/src/x509.rs

```diff
@@ -333,35 +311,56 @@ mod test {
 
     #[test]
     fn verify_valid_cert_chain() {
-        let cert_chain = UnverifiedCertChain::try_from_pem([LEAF_CERT, PROCESSOR_CA, ROOT_CA])
-            .expect("failed to parse cert chain");
+        let time = DateTime::from_unix_duration(Default::default())
+            .expect("failed to get UNIX EPOCH time");
+        let chain = [LEAF_CERT, PROCESSOR_CA, ROOT_CA]
+            .iter()
+            .map(|cert| X509Certificate::from_pem(cert).expect("failed to parse cert"))
+            .collect::<Vec<_>>();
         let trust_anchor = TrustAnchor::try_from_pem(ROOT_CA).expect("failed to parse root cert");
-        let crl = CertificateRevocationList::try_from_pem([ROOT_CRL, PROCESSOR_CRL])
-            .expect("failed to parse CRL");
-        assert!(cert_chain.verify(&trust_anchor, crl).is_ok());
+        let crls = [ROOT_CRL, PROCESSOR_CRL]
+            .iter()
+            .map(|crl| CertificateList::from_der(crl).expect("failed to parse CRL"))
+            .collect::<Vec<_>>();
+        let verifier = X509CertificateChainVerifier::new(trust_anchor);
+        assert!(verifier
+            .verify_certificate_chain(chain.iter(), crls.iter(), time)
```

Comment:
Does the time here actually matters?

Reply:
nope, i missed this when I did some refactoring cleanups to use a constant `DateTime` value.  Will fix to jive with the other test cases.

Reply:
updated now